src/styles/blocks/card.scss

```diff
@@ -0,0 +1,85 @@
+.card {
```

Comment:
your card should have padding for all sides to avoid adding margin-left/right to children